components/composed/collection/CollectionUpdateForm/CollectionUpdateForm.tsx

```diff
@@ -56,18 +68,18 @@ export default function CollectionUpdateForm({ data, onSuccess }: Props) {
           ]}
           {...register("visibility")}
         />
-        <Forms.Textarea
-          label="forms.collection.fields.summary"
-          {...register("summary")}
-        />
-        <Forms.Datepicker
-          label="forms.collection.fields.visibleAfterAt"
-          {...register("visibleAfterAt")}
-        />
-        <Forms.Datepicker
-          label="forms.collection.fields.visibleUntilAt"
-          {...register("visibleUntilAt")}
-        />
+        <Forms.HiddenField watch={watch} field="visibility" showOn="LIMITED">
```

Comment:
`HiddenField` takes the form's `watch` function, a field to watch, and a value to check. If the field equals the value passed in, the child fields show!